lms/djangoapps/teams/static/teams/templates/team-member.underscore

```diff
@@ -1,7 +1,7 @@
 <li>
   <span class="team-member">
     <a class="member-profile" href="<%= memberProfileUrl %>">
```

Comment:
You can do this later, but all `<%=` should be `<%-`, unless it contains an HtmlUtils function.